src/audio.c

```diff
@@ -1304,7 +1311,7 @@ static int autx_print_pipeline(struct re_printf *pf, const struct autx *autx)
 }
 
 
-static int aurx_print_pipeline(struct re_printf *pf, const struct aurx *rx)
+static int aurx_print_pipeline(struct re_printf *pf, struct aurx *rx)
```

Comment:
we should try to avoid this

with this change, we are loosing the "const-ness".
removing const means that the function is modifying the state.

this was working with the old "struct lock", why is it not working with "mtx_t" ?

Reply:
`mtx_alloc` is now available: https://github.com/baresip/re/commit/79a6168f144125da99b0173f96cfda20b7e925b8

Reply:
this is great !

we should try to keep the const-ness, it gives a hint to the compiler about what
can be optimized, and it also improves readability.

I think there are 1-2 places in re+rem that also has the same issue with const and mtx_t